extensions/reaper/manifest.json

```
@@ -31,10 +31,10 @@
   },
   "extension": {
     "name": "reaper",
-    "title": "Reaper",
-    "description": "A view to a kill.",
+    "title": "Process monitor",
```

```suggestion
    "title": "Process Monitor",
```

I think most of the titles of gallery content are in title case, so I'm updating this to match.